.github/workflows/publish.yml

```diff
@@ -6,9 +6,20 @@ on:
     tags: ['*']
 
 jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
+        with:
+          module-name: 'queue'
+
   publish-rockspec-scm-1:
     if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
+    needs: version-check
```

Comment:
Let's remove it here since it doesn't do anything

Reply:
Deleted